### PR TITLE
Added a config option to show showcases in search results

### PR DESCRIPTION
**Pull request by roll**
- connects https://github.com/okfn/ckanext-lacounts/issues/28
- followed by https://github.com/okfn/docker-ckan-lacounts/pull/3

### CODE REVIEW

**Comment by roll** [comment]
@amercader 
please take a look

File: README.rst [hunk 1]
```diff
@@ -78,6 +78,13 @@ do::
     python setup.py develop
     pip install -r dev-requirements.txt
 
+---
+Configuration
+---
+
+- `ckan.showcase.include_into_search (bool)` - include showcases to datasets search
```
Comment (amercader) [inline README.rst]: Can you rename to `ckanext.showcase.include_in_search`. and `disqus.name` (1 s)

File: ckanext/showcase/plugin.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import ckan.plugins as plugins
 import ckan.lib.plugins as lib_plugins
 import ckan.lib.helpers as h
+from ckan.common import config
```
Comment (amercader) [inline ckanext/showcase/plugin.py]: Move to existing import on line 10

**Review by amercader — APPROVED** [review]
Looks good, 2 minor changes

**Comment by roll** [comment]
thanks @amercader, fixed

**Pull request merged**